index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/css2?family=Do+Hyeon&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <title>Tip Calculator</title>
+</head>
+<body>
+
+    <!-- Codepen can be found here: https://codepen.io/ken-taylor-codes/pen/NWqmWjj -->
+
+    <header>
+        <h1>Tip Calculator</h1>
+        <p>Enter the cost of your meal and your desired tip and we will calculate your tip and total cost!</p>
+    </header>
+
+    <section>
```

Comment:
Great use of semantic HTML. Everything is easy to follow and names are expressive.